### PR TITLE
Improve name in installation and branch management in update

### PR DESCRIPTION
**Pull request by poupotte**

### CODE REVIEW

File: lib/main.coffee [hunk 1]
```diff
@@ -197,11 +199,11 @@ program
 
 # Update
 program
-    .command("update <app> [repo]")
+    .command("update <app>")
     .description(
         "Update application (git + npm) and restart it. The 'repo' option " +
```
Comment (frankrousseau) [inline lib/main.coffee]: Description must be updated since `repo` is no more an option.

File: lib/stack_application.coffee [hunk 2]
```diff
@@ -39,6 +39,32 @@ msgRepoGit = (app) ->
...
+    manifest = JSON.parse(fs.readFileSync jsonPackage, 'utf8')
+
+    # Retrieve manifest from git config
+    config = fs.readFileSync configGit, 'utf8'
+    # Retrieve repository url
+    manifest.repository =
+        'url': config.split('url = ')[1].split('\n')[0]
+        'type': 'git'
+    # Retrieve branch
+    for part in config.split('[')
```
Comment (frankrousseau) [inline lib/stack_application.coffee]: Can you add comments to explain what you do (and why) in this for loop?

**Comment by poupotte** [comment]
This PR contains change-branch function.
Wait new cozy-clients version to upgrade it !

**Comment by frankrousseau** [comment]
No worry the lib won't be packaged until cozy-clients is ready.

**Pull request merged**